.eslintrc.json

```diff
@@ -17,8 +17,20 @@
           "node/no-process-env": 0,
           "node/no-process-exit": 0,
           "node/no-sync": 0,
+          "arrow-spacing": ["error", {"before": true, "after": true}],
+          "brace-style": ["error", "1tbs"],
+          "comma-dangle": ["error", "always-multiline"],
+          "curly": ["error", "all"],
+          "eol-last": ["error", "always"],
+          "indent": ["error", 2, {"SwitchCase": 1, "ObjectExpression": "first"}],
           "quotes": ["error", "single"],
-          "semi": ["error", "never"]
+          "semi": ["error", "never"],
```

Comment:
If we're not using semicolon, we could have this rule present: https://eslint.org/docs/latest/rules/no-unexpected-multiline

Reply:
I'd personally prefer semis but went with what was most prevalent already ;)

Good one though; adding it as well!